.env.example

```
@@ -0,0 +1,63 @@
+# The server container also receives DATABASE_URL from docker-compose.yml (postgres hostname).
+
+# ── Database (required) ────────────────────────────────────────────────
+# Non-Docker / host Node: keep localhost here. Compose overrides this inside the API container.
+DATABASE_URL=postgresql://postgres:postgres@localhost:5432/internhack
+
+# ── Server (minimum boot: DATABASE_URL + JWT_SECRET in server/src/index.ts) ──
+JWT_SECRET=
+
+DODO_PAYMENTS_API_KEY=
+DODO_PAYMENTS_WEBHOOK_KEY=
+DODO_PAYMENTS_ENVIRONMENT=live_mode
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DODO_PAYMENTS_ENVIRONMENT key should go before the DODO_PAYMENTS_WEBHOOK_KEY key

(UnorderedKey)